### DISCUSSION TITLE
Getting black screens

### DISCUSSION THREAD

**Discussion by quintendewilde** [deforum-art/sd-webui-text2video#134] [Q&A]
I'm using Google Colab for running this and my prompt is: woman dancing

I got it to work, but now I only get black screens.
Is this the NSFW filter acting strange?

Can I do something about it?

**Comment by kabachuha** [comment]
Try it in non half-vae mode

**Reply by quintendewilde** [reply]
In txt2video I don't have that option

**Reply by quintendewilde** [reply]
Or do you mean in the python code?

**Reply by kabachuha** [reply]
See the 'Settings' tab to the right and proceed to the 'text2video' section below

**Reply by quintendewilde** [reply]
I will try it out

**Reply by quintendewilde** [reply]
@kabachuha do I just set false next to it?

**Reply by quintendewilde** [reply]
As in I see no buttons
<img width="2156" alt="Schermafbeelding 2023-04-22 om 17 57 29" src="https://user-images.githubusercontent.com/6031569/233794518-74a36537-b4d4-4642-bc14-6ad2314dea2b.png">
